### PR TITLE
Add flatten command

### PR DESCRIPTION
**Pull request by balinthaller**

### CODE REVIEW

File: carte_cli/utils/templates/dataset_flattened.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# {{ dataset.name }}
```
Comment (balinthaller) [inline carte_cli/utils/templates/dataset_flattened.md]: This template basically has all the same classes and HTML elements as the MkDocs template override we've been using internally.

File: carte_cli/utils/templates/dataset_flattened.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# {{ dataset.name }} {: .page-title }
```
Comment (balinthaller) [inline carte_cli/utils/templates/dataset_flattened.md:1]: The `{: ... }` tags are Markdown attribute lists, they control how what HTML attributes the compiled markdown will have. More on this: https://python-markdown.github.io/extensions/attr_list/
We can't use straight HTML as that breaks search results in weird ways (mostly likely during term tokenisation).

File: carte_cli/main.py [hunk 3]
```diff
@@ -58,5 +59,19 @@ def new_frontend(
     create_frontend_dir(name, init_admin=(not no_admin), sample_data=(not no_sample))
 
 
+@app.command("flatten")
+def flatten(
+    input_dir: str = typer.Argument(
+        ..., help="The source metadata directory, the same as the extraction output"
```
Comment (IstvanM) [inline carte_cli/main.py:65]: what is the `...`?

Comment (balinthaller) [reply carte_cli/main.py:65]: Special value in Python (haven't encountered it before this either). In Typer means the argument is required.

**Review by IstvanM — APPROVED** [review]

**Pull request merged**